tests/e2e/support/page/files/version.ts

```diff
@@ -28,4 +39,33 @@ export class VersionPage {
     const elements = this.fileVersionSidebarXPath
     return await elements.count()
   }
+
+  async restoreOlderVersion({ files, folder }: { files: File[]; folder?: string }) {
+    const { page } = this.actor
+    const { dir: resourceDir, base: resourceBase } = path.parse(folder)
+    const fileName = files.map((file) => path.basename(file.name))
+    await filesCta.navigateToFolder({ page: page, path: folder })
+    await filesCta.sidebar.open({ page: page, resource: fileName[0] })
+    await filesCta.sidebar.openPanel({ page: page, name: 'versions' })
+    const fileSize = await this.fileVersionSizeXPath.textContent()
+    VersionPage.olderVersionSize = fileSize.trim()
+    await Promise.all([
+      page.waitForResponse(
+        (resp) =>
+          resp.url().includes('/v/') && resp.status() === 204 && resp.request().method() === 'COPY'
+      ),
+      await this.fileVersionRevertButtonXPath.click()
+    ])
+    // Files details page does not update after restore button is clicked
+    await page.reload()
+  }
+
+  async checkOlderVersionRestored({ resource }: { resource: string }) {
+    const { page } = this.actor
+    this.fileSizeXPath = this.actor.page.locator(
+      `//*[@data-test-resource-name="${resource}"]/ancestor::tr//*[@class="oc-resource-size"]`
+    )
+    const fileSize = await this.fileSizeXPath.textContent()
+    expect(fileSize.length).toBe(VersionPage.olderVersionSize.length)
```

Comment:
Nice job) You compared: 
this
<img width="827" alt="Screenshot 2022-02-07 at 22 10 03" src="https://user-images.githubusercontent.com/84779829/152872624-b55b6348-e96f-4b6f-b0b1-ed1e462b43be.png">


with this:
<img width="836" alt="Screenshot 2022-02-07 at 22 10 35" src="https://user-images.githubusercontent.com/84779829/152872612-1cd969a6-c794-4c11-ae1c-42108dbcb676.png">

When we create new version, we add the same file (same length). Can you fix it please? 
Use 2 files of different lengths.

for example, change here:
```
When "Alice" creates a new version of the following files
      | resource                   | to               |
      | PARENT/simple.pdf | folder_to_shared |
```

Reply:
Fixed on the recent commit.